scss/_patterns_switch.scss

```
@@ -29,27 +35,27 @@ $knob-size: $sp-unit * 3;
   }
 
   .p-switch__slider {
-    @extend %vf-has-round-corners;
-
-    background: linear-gradient(to right, $color-link 50%, $color-mid-light 50%);
+    background: $color-mid-light;
+    border-radius: $knob-size * 1.5;
     box-shadow: inset 0 2px 5px 0 transparentize($color-dark, 0.8);
     display: inline-block;
-    height: $knob-size;
+    height: $knob-size * 2;
```

`$knob-size` variable should already have the value needed here, as in comment above.

```suggestion
    height: $knob-size;
```